schema.graphql

```diff
@@ -16,12 +16,23 @@ type ThirdParty @entity {
 }
 
 type Item @entity {
+  # URN
   id: ID!
-  collection: String!
+  metadata: Metadata!
```

Comment:
I would add a new property `itemBlockchainId` with the real id of the item. It means the one that comes in the event. The `urn`, and the `urnSuffix` for the item identifier itself inside the collection.

Also, what do you think of using the `ID` with the same value of the `urn` ?

Reply:
Changed to have the same value of the URN.